tools/cloudflow-operator/src/main/scala/cloudflow/operator/action/Actions.scala

```diff
@@ -34,59 +37,45 @@ object Actions {
    * The application data is kept in 0-S savepoints.
    */
   def deploy(
-      newApp: CloudflowApplication.CR,
-      currentApp: Option[CloudflowApplication.CR] = None,
+      newApp: App.Cr,
+      currentApp: Option[App.Cr] = None,
       runners: Map[String, runner.Runner[_]],
       podName: String,
       podNamespace: String,
-      cause: ObjectResource
-  ): Seq[Action] = {
+      cause: ObjectReference): Seq[Action] = {
     require(currentApp.forall(_.spec.appId == newApp.spec.appId))
-    val labels          = CloudflowLabels(newApp)
-    val ownerReferences = CloudflowApplication.getOwnerReferences(newApp)
+    val labels = CloudflowLabels(newApp)
+    val ownerReferences = List(Util.getOwnerReference(newApp.name, newApp.getMetadata.getUid))
```

Comment:
Rather create an object with apply called AppOwnerReference, so this becomes:
```
val ownerReferences = List(AppOwnerReference(newApp.name, newApp.getMetadata.getUid))
```
`Util` class pet peeve triggered ;-)

Reply:
I understand, it was a stub 👍  removed